torch/_inductor/lowering.py

```diff
@@ -311,9 +350,24 @@ def wrapped(*args, **kwargs):
             assert not any(
                 x == "out" for x in kwargs.keys()
             ), "out= ops aren't yet supported"
-
+        unsupported_symbolify_lowerings = [
+            "index_put",
+            "_unsafe_index_put_",
+            "where",
+            "flex_attention",
+        ]
```

Comment:
These lowering rules are the main reasons for ci errors and I found they are hard to fix when they involve symbol arguments. The easiest way is to skip them. not sure if it is the proper way.

Reply:
imo it's better to leave the ci errors in that add something we can't land because we can at least look at where ci went wrong